BitwardenShared/UI/Vault/Vault/VaultList/VaultListProcessor.swift

```diff
@@ -289,6 +312,26 @@ extension VaultListProcessor {
         state.profileSwitcherState.isVisible = visible
     }
 
+    /// Triggers the app review prompt after a delay.
+    private func scheduleReviewPrompt() async {
+        reviewPromptTask?.cancel()
+        reviewPromptTask = Task {
+            do {
+                try await Task.sleep(nanoseconds: Constants.appReviewPromptDelay)
+                if await services.configService.getFeatureFlag(.appReviewPrompt) {
+                    state.isEligibleForAppReview = true
+                }
+                if await services.configService.getFeatureFlag(.enableDebugAppReviewPrompt) {
+                    state.toast = Toast(title: Constants.appReviewPromptEligibleDebugMessage)
+                }
```

Comment:
🤔 Sorry but I'm not understanding: why is this feature flag necessary and why is it showing the toast? Given that you could already control the original `.appReviewPrompt` flag on the Debug menu as well.

Reply:
This update is primarily for ```PM-15531```. The system allows the review prompt to be displayed to a user a maximum of three times within a 365-day period. To simplify testing, especially for TestFlight builds, we are showing a toast message when the user is eligible for the app review prompt.